product_variant_default_code/i18n/pt_BR.po

```diff
@@ -201,4 +201,4 @@ msgid "[attribute3]-[attribute1]"
 msgstr "[attribute3]-[attribute1]"
 
 #~ msgid "Product Template"
-#~ msgstr "Produto Modelo"
+#~ msgstr "Produto Modelo"
```

Comment:
@BhaveshHeliconia it looks you touch these 3 translations file and just removed the last line for no reason. Probably not a blocker, but I'm not too sure either it's okay. If you fix that I'll merge.